docs/semgrep-assistant/analyze.md

```diff
@@ -0,0 +1,45 @@
+---
+slug: analyze
+title: Analyze findings
+hide_title: true
+description: Learn how to analyze findings with Assistant and view its results.
+tags:
+  - Deployment
+  - Semgrep Assistant
+---
+
+# Analyze findings with Semgrep Assistant
+
+Once you've [enabled Assistant](/docs/semgrep-assistant/getting-started), you can use the **Analyze** button on the [Findings page](/semgrep-code/findings) to trigger all Assistant functions, including autofix, auto-triage, and component tagging, on existing findings.
+
+![Assistant Analyze button on Findings page](/img/scp-assistant.png#md-width)
+
+To analyze your findings with Assistant:
+
+1. On the [Findings](https://semgrep.dev/orgs/-/findings?tab=open) page, select the findings that you want Assistant to analyze.
+2. Click **Analyze**.
+3. In the confirmation window that appears, confirm that you want to analyze your findings with Assistant.
```

Comment:
```suggestion
3. In the confirmation dialog that appears, confirm that you want to analyze your findings with Assistant.
```

I think this is a dialog because it's not modular (https://developers.google.com/style/ui-elements#term-windows)